types/domutils/index.d.ts

```diff
@@ -0,0 +1,129 @@
+// Type definitions for domutils 1.7
+// Project: https://github.com/FB55/domutils#readme
+// Definitions by: Johan Davidsson <https://github.com/johandavidson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+import { DomElement } from "domhandler";
+export namespace DomUtils {
```

Comment:
This seems wrong to me? Doesn't the source library export all of these [at the top level](https://github.com/fb55/domutils/blob/master/index.js)?